apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,788 @@
+<template>
+  <Base
+    :show-search="true"
+    :title="curr_title"
+    @changed-files="evalInfo = null"
+  >
+    <!-- Topbar content - give it a search bar -->
+    <template #topbar-content>
+      <v-btn :disabled="!can_clear" @click="clear">
+        <span class="d-none d-md-inline pr-2"> Clear </span>
+        <v-icon>mdi-filter-remove</v-icon>
+      </v-btn>
+      <UploadButton />
+      <div class="text-center">
```

Comment:
please make the export button a 'generic' component that all of the views can leverage.  just pass in the names/functionality for what exports you actually want there.  this is part of the effort to modularize this view so that it's not multi-hundred lines long.

Reply:
Also can you fix the naming in uploadbutton.vue so that it has js conventions.

Reply:
Fixed the UploadButton naming and creating an ExportButton generic component to be more modular